odfdom/src/test/java/org/odftoolkit/odfdom/doc/table/TableCellTest.java

```diff
@@ -234,34 +235,35 @@ public void testGetSetTextValue() throws Exception {
     Assert.assertEquals("Aabbccddee", text);
   }
 
-  @Test @Ignore // FIXME test failure: Expected: #0.0 Actual: 0.0
+  @Test @Ignore // Unsupported field: MonthOfYear
+    // # is an optional integer, like format #.0 with value 0.3 shows .3
   public void testSetGetFormat() throws Exception {
     OdfSpreadsheetDocument odsdoc = loadInputOds();
 
     int rowindex = 3, columnindex = 0;
     OdfTable table = odsdoc.getTableByName("Sheet1");
     OdfTableCell fcell = table.getCellByPosition(columnindex, rowindex);
 
-    fcell.setFormatString("#0.0");
+    fcell.setFormatString("#.0");
     String displayvalue = fcell.getDisplayText();
     Assert.assertEquals(
         "300" + (new DecimalFormatSymbols()).getDecimalSeparator() + "0", displayvalue);
     String format = fcell.getFormatString();
-    Assert.assertEquals("#0.0", format);
+    Assert.assertEquals("#.0", format);
 
     OdfTableCell dcell = table.getCellByPosition(3, 2);
     format = dcell.getFormatString();
-    Assert.assertEquals("MMM d, yy", format);
+    Assert.assertEquals("D. MMM YY", format);
```

Comment:
You used the german format with a day-of-year and week-based-year here. I think this should be `d. MMM yy`. The old code used the american format. Which one to use depends on the input spreadsheet and maybe the locale setting, but I have to check the spreadsheet first.